gateway/src/apicast/policy/headers/headers.lua

```diff
@@ -5,6 +5,8 @@
 -- Similarly, this policy also allows to add, modify, and delete the headers
 -- included in the response.
 
+local tablex = require('pl.tablex')
```

Comment:
I really don't want to depend on penlight in policies. OpenResty provides table.clone that does shallow clone. 

Also, if we are doing this, then it should be pushed from the top, not done in each policy.

Reply:
OK. I suggest we solve the problem in the loader first and leave the immutable config for a future PR.

Reply:
Agreed 👍